lib/Ix/Util.pm

```diff
@@ -62,6 +63,8 @@ sub differ ($x, $y) {
 
   return $x ne $y if ! ref $x;
 
+  return $x ne $y if is_bool($x) && is_bool($y);
```

Comment:
The correct operator for boolean difference is `xor`.  This is a bit of a nit-pick, but consider that when we changed JSON libraries, some booleans started to stringify as `false` instead of `0`.  If we had two false booleans that stringified differently, this would mark them different.  This shouldn't happen, but we will know we're safe if we use `xor`.